packages/components/workbench/src/WorkbenchHeader/WorkbenchHeader.tsx

```diff
@@ -18,7 +18,7 @@ export interface WorkbenchHeaderProps extends CommonProps {
   /** This is the icon that will be shown on the left side of the title and it's possible to use Forma 36’s icons or external icons */
   icon?: IconComponent | ReactElement;
   /** This is the text that will be shown on the right side of the title in the Header component */
-  description?: string;
+  description?: string | ReactElement;
```

Comment:
it should be `ReactNode`, if it's `ReactElement` then it will have to be rendered like `<Description>` (like icon prop), but we don't need it for the description
```suggestion
  description?: string | ReactNode;
```